### PR TITLE
Add new buttons to landing page

### PR DESCRIPTION
**Pull request by Aerilym**
Screenshot from 2024-06-03 17-45-00
Screenshot from 2024-06-03 17-40-05

### CODE REVIEW

File: components/HomeHero.tsx [hunk 1]
```diff
@@ -3,13 +3,57 @@ import { EXCHANGES, UI } from '@/constants';
...
+    ariaLabel: 'Link to Oxen Coin Claims Program',
+  },
+] as const;
+
 export function HomeHero() {
   return (
     <div
       className={classNames(
-        'relative w-full flex justify-center items-center overflow-x-hidden',
+        'relative w-full flex justify-center items-center overflow-x-hidden pt-96',
```
Comment (yougotwill) [inline components/HomeHero.tsx:54]: Slightly overkill but you can't see the heading properly at 320px. Don't worry about the button overflow but at least the content should be readable 🙏

Screenshot 2024-06-04 at 09 59 39

**Review by yougotwill — CHANGES_REQUESTED** [review]

File: components/HomeHero.tsx [hunk 2]
```diff
@@ -3,13 +3,57 @@ import { EXCHANGES, UI } from '@/constants';
...
+  href: string,
+  ariaLabel: string,
+  label: string,
+}
+
+function HeroButton({
+  href,
+  ariaLabel,
+  label,
+}:HeroButtonProps) {
```
[the remark this answers is not in the document]
Comment (Aerilym) [reply components/HomeHero.tsx:16]: Yeah thats weird, fixed

File: components/HomeHero.tsx [hunk 3]
```diff
@@ -3,13 +3,57 @@ import { EXCHANGES, UI } from '@/constants';
...
+      <Button size="large" className="font-sans">
+        {label}
+      </Button>
+    </a>
+  );
+}
+
+const heroButtons: Array<HeroButtonProps> = [
+  {
+    href: 'https://oxen.io/blog/development-is-transitioning-to-session-token/',
```
[the remark this answers is not in the document]
Comment (Aerilym) [reply components/HomeHero.tsx:34]: Good catch, fixed

File: components/HomeHero.tsx (hunk 1 above)
Comment (Aerilym) [reply components/HomeHero.tsx]: Redone styles

**Review by yougotwill — APPROVED** [review]
LGTM! 🚢

**Pull request merged**